docs/source/platform/schema-registry.md

```diff
@@ -177,3 +166,12 @@ const server = new ApolloServer({
   }
 });
 ```
+
+<h2 id="benefits">Tools that use the schema registry</h2>
+
+An up-to-date schema in Apollo's schema registry enables a number of workflow benefits through integrations with other tools. A registered schema combined with:
```

Comment:
How about asking community members to make a PR to add another tool that leverages the schema registry? I know no one has done it yet, but maybe advertising it like that will inspire someone to :)

Reply:
This whole section feels a little out of place, like something's missing. I'm not sure what direction to take it in though.